src/ui/pages/SendTransaction/NetworkFee/NetworkFeeDialog.tsx

```diff
@@ -178,8 +178,8 @@ function CustomNetworkFeeForm({
     }
 
     return {
-      priorityFeeFiat: getFiatValue(
-        (priorityFee || 0) + (eip1559?.base_fee || 0)
+      expectedFeeFiat: getFiatValue(
+        Math.min((priorityFee || 0) + (eip1559?.base_fee || 0), maxFee)
```

Comment:
This can be defined as variable. Cause as I can see, we will use the same formula below.

Reply:
But the values are in different scopes?